featuretools/primitives/standard/transform/__init__.py

```diff
@@ -0,0 +1,31 @@
+# flake8: noqa
```

Comment:
same here

Reply:
Same issue - the `import *` causes linting problems

Reply:
got it